{
    "checksum": "f59963b2dd17a19e80c636a0089d90c06402f10e6c7d22ebf4ae3876cc041417",
    "file": "error.py",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/x-python"
    },
    "number_of_duplicates": 1,
    "package": "pyzmq",
    "path": "pyzmq/22.3.0-1/zmq/error.py",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/pyzmq/22.3.0-1/",
        "metric": {
            "size": 2344
        },
        "pts_link": "https://tracker.debian.org/pkg/pyzmq",
        "sloc": [
            [
                "python",
                14864
            ],
            [
                "ansic",
                1002
            ],
            [
                "cpp",
                404
            ],
            [
                "makefile",
                171
            ],
            [
                "sh",
                89
            ]
        ],
        "suites": [
            "bookworm",
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/python-team/packages/pyzmq",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/p/pyzmq/22.3.0-1/zmq/error.py",
    "stat": {
        "perms": "rw-r--r--",
        "size": 5458,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "22.3.0-1"
}